.pre-commit-config.yaml

```diff
@@ -17,3 +17,4 @@ repos:
     rev: f295829140d25717bc79368d3f966fc1f67a824f  # frozen: v0.41.0
     hooks:
       - id: markdownlint
+exclude: SECURITY.md
```

Comment:
> > It breaks pre-commit checks.
> 
> I will exclude the SECURITY.md file from pre-commit checks since this file is managed by the security team and should not be modified.

I disagree with this approach:
Excluding SECURITY.md from pre-commit checks (markdowlint) means that some Markdown files will have rules enforced, and some will not. 

This breaks the codebase's coherence, adds complexity, and complicates contributors' mental models. With such a change, each contributor has to remember, "Ah yeah, this file uses these rules, this file does not." It should be avoided as much as possible and used as a last resort, IMHO.

Also excluding it totally means that nothing will be checked in it (including other rules of markdownlint). I don't think it's desirable.

I see two acceptable options:
* Either the SECURITY.md is updated so that it can pass the checks
* Either the pre-commit checks (config) need to be adapted to allow longer line length. (this requires updating it in all repositories to keep coherence, I would avoid this as much as possible and first consider changing the content of SECURITY.md)


Can you please format this `SECURITY.md` according to our standards (pre-commit config) and **get in touch with security team** (please do) asking them if using this format is acceptable for them (using shorter line length) and explaining why (pre-commit validation we have in place in most of our repositories requires line length of maximum: x chars, 1 blank line after titles etc. (all changes not in conformity with pre-commit checks -> markdownlint).